src/target/stm32f4.c

```diff
@@ -160,7 +160,10 @@ static void stm32f4_add_flash(target *t,
 	f->length = length;
 	f->blocksize = blocksize;
 	f->erase = stm32f4_flash_erase;
-	f->write = stm32f4_flash_write;
+	f->write = target_flash_write_buffered;
```

Comment:
Reference manuals for F2, F4 and F7 all seem to explicitly indicate you can do repeated write cycles so long as you only set 1s to 0s.  Unless something here is actually broken I see no need to change it.